native/src/components/NavigationTiles.tsx

```diff
@@ -27,12 +27,12 @@ const TilesRow = styled.View`
   shadow-offset: 1px;
 `
 
-type PropsType = {
+type NavigationTilesWithScrollableViewProps = {
```

Comment:
should be NavigationTilesProps

Reply:
This is a case where the file name and the name written in the export differs (export default NavigationTilesWithScrollableView). Should I always chose the file name for the props renaming? Or should I rename the export name to the file name?

Reply:
I think best here would be to also rename the component and the export. Do not know why this should be named like it currently is. (@steffenkleinle do you see a reason to keep naming as it is now?)

Not part of this ticket, but would be nice if you can do it.

Reply:
No, I think we are good to rename it, `NavigationTiles` is just fine.